src/extension.ts

```diff
@@ -8,11 +8,14 @@ import { TestCaseRecorder } from "./testUtil/TestCaseRecorder";
 import { getCommandServerApi, getParseTreeApi } from "./util/getExtensionApi";
 import isTesting from "./testUtil/isTesting";
 import CommandRunner from "./core/commandRunner/CommandRunner";
+import updateMissingSetting from "./util/updateMissingSetting";
 
 export async function activate(context: vscode.ExtensionContext) {
   const { getNodeAtLocation } = await getParseTreeApi();
   const commandServerApi = await getCommandServerApi();
 
+  updateMissingSetting("editor.padding.top", 5);
```

Comment:
Hesitant to add to this file, as we're currently trying to remove as much as we can. Maybe this belongs in init function of `Decorations`?

Reply:
Even though this helps with decoration problem something as core as updating the user settings I still believe belongs in this file. In the case we have to add another setting it's nice to have all the changes we make to the user setting in one place.

Reply:
Ok, I don't feel super strongly so let's leave it here